modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/client/Wso2EventServer.java

```diff
@@ -153,9 +148,9 @@ public void run() {
         try {
             startServer();
         } catch (DataBridgeException e) {
-            e.printStackTrace();
+            log.error(e.getMessage(), e);
```

Comment:
This is not a correct way of logging.  Add a proper log message and stack-trace. Check other places as well.